docs/source/install_tutorials/osx_11_5_1.rst

```diff
@@ -295,7 +295,7 @@ Step 8: Test Hagrid (optional)
 
     |hagrid_startup_complete|
 
-    You can then load "http://localhost:8081" to see a UI deployed which looks like:
+    You can then load "http://localhost:80" to see a UI deployed which looks like:
```

Comment:
By default , when you launch hagrid without any custom port , it defaults to `8081` port.

Reply:
thats correct